threenums.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>   /*Armstrong Threenums it checks if all three numbers are equal, if they are not it gives you the largest number*/
+int main()
+{
+	printf("choose a number\n");
+	int x;
+	scanf("%d", &x);
+	printf("choose another number\n");
+	int y;
+	scanf("%d", &y);
+	printf("choose a final  number\n");
+	int z;
+	scanf("%d", &z);
+	if ((x==y)&&(x==z)&&(y==z)){
+	printf("all the numbers are equal\n");}
+	else{
+	printf("the numbers are not all equal\n");
+		if ((x>y)&&(x>z)){
```

Comment:
nice job, but when you input 5, 3, 5 in that order it doesn't output the largest number. looking at your code, I'm pretty sure that's because you didn't consider the cases where two numbers are equal and one isn't.